test/gtest/uct/test_ud_slow_timer.cc

```diff
@@ -44,6 +44,15 @@ class test_ud_slow_timer : public ud_base_test {
     {
         return UCS_ERR_INVALID_PARAM;
     }
+
+    void wait_for_rx_sn(unsigned sn)
+    {
+        ucs_time_t deadline = ucs_get_time() +
+                              ucs_time_from_sec(10) * ucs::test_time_multiplier();
```

Comment:
minor: we have `UCT_TEST_TIMEOUT_IN_SEC` and `DEFAULT_TIMEOUT_SEC` macro in uct_test.h

Reply:
i cherry picked the fix as is from master..